tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/PublicAssetGetMethodContract.scala

```diff
@@ -0,0 +1,310 @@
+        s"""{
+           |	"sessionState": "${SESSION_STATE.value}",
+           |	"methodResponses": [
+           |		[
+           |			"error",
+           |			{
+           |				"type": "unknownMethod",
+           |				"description": "Missing capability(ies): com:linagora:params:jmap:public:assets"
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin))
+
+
+  @Test
+  def getShouldReturnEmptyAssetsByDefault(): Unit =
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:public:assets"],
+           |	"methodCalls": [
+           |		[
+           |			"PublicAsset/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": null
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("methodResponses[0]", jsonEquals(
+        s"""[
+           |    "PublicAsset/get",
+           |    {
+           |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |        "state": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
+           |        "list": [],
+           |        "notFound": []
+           |    },
+           |    "c1"
+           |]""".stripMargin))
+
+  @Test
+  def fetchNullIdsShouldReturnAllAssets(server: GuiceJamesServer): Unit = {
+    val publicAsset = server.getProbe(classOf[PublicAssetProbe]).create(BOB, CREATION_REQUEST)
+    val publicAsset2 = server.getProbe(classOf[PublicAssetProbe]).create(BOB, CREATION_REQUEST)
+
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:public:assets"],
+           |	"methodCalls": [
+           |		[
+           |			"PublicAsset/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": null
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("methodResponses[0]", jsonEquals(
+        s"""[
+           |    "PublicAsset/get",
+           |    {
+           |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |        "state": "$${json-unit.ignore}",
+           |        "list": [
+           |                  {
+           |                      "id": "${publicAsset.id.value}",
+           |                      "publicURI": "${publicAsset.publicURI.value}",
+           |                      "size": 3,
+           |                      "contentType": "image/png",
+           |                      "blobId": "${publicAsset.blobId.asString()}",
```

Comment:
As said, I am not fully sure the blobId shall be exposed in JMAP /get.

Could we remove it for now as:
 - we do not have blobResolvers for this
 - I do not see a use case where we want the blobId and the public URL wouldn't be enough...
 
 Also remember to update specs accordingly...
 
 ```suggestion
```